packages/babel-plugin-block/macro.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * External dependencies
+ */
+const { createMacro } = require( 'babel-plugin-macros' );
+const { existsSync, readFileSync } = require( 'fs' );
+const { mapKeys, pick } = require( 'lodash' );
+const { dirname, join, relative } = require( 'path' );
+
+const i18nVariable = '_x';
```

Comment:
I don't imagine there's much value in making this configurable, since `_x` specifically assumes to be used with an additional argument for the context, which in this implementation is hard-coded and not-configurable.

Reply:
I agree I don't remember why I introduced it :)